### PR TITLE
TS transform plugin README updatex2

### PR DESCRIPTION
**Pull request by orta**
Took in the extra feedback and updated the doc again.

### CODE REVIEW

File: packages/babel-plugin-transform-typescript/README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Does not type-check its input. For that, you will need to install and set up Typ
...
 * Does not support [`namespace`][namespace]s.
-* Does not support [`const enum`][const_enum]s because those require type information to transpile.
-* Does not support [`export =`][exin] and [`import =`][exin], because those cannot be transpiled to ES.next.
+* Does not support [`const enum`][const_enum]s because those require type information to compile.
+* Does not support [`export =`][exin] and [`import =`][exin], because those cannot be compile to ES.next.
 
 ## Workarounds
 
-* `namespace`: Migrate to using the `module { }` syntax instead.
+* `namespace`: Move to using [file exports][fm], or migrate to using the `module { }` syntax instead.
```
Comment (existentialism) [inline packages/babel-plugin-transform-typescript/README.md]: Thoughts on combining the workaround next to its caveat?

Comment (orta) [reply packages/babel-plugin-transform-typescript/README.md]: works for me

**Comment by orta** [comment]
How does this feel?

**Comment by existentialism** [comment]
Works for me, maybe later someone can spruce it up with examples (not high priority), thanks @orta!

**Pull request merged**